MapboxCoreNavigation/PassiveLocationDataSource.swift

```diff
@@ -46,6 +46,13 @@ open class PassiveLocationDataSource: NSObject {
      */
     let navigator: Navigator
     
+    /**
+     Whether the navigator’s router has been configured.
+     
+     Set this property to `true` before calling `Navigator.configureRouter(for:)` and reset it to `false` if something causes the router to be unconfigured.
```

Comment:
Apparently there’s no way to ask the navigator whether the router has been configured. Fortunately, there’s also no situation where something else might call `configureRouter(for:)` on our behalf, making this property go stale, and we don’t currently call `removeTilesForPackedTilesPath(_:lowerLeft:upperRight:)` or `clearCache()`.